server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -177,6 +186,35 @@ const orderMutations = {
         order = await order.update({ PaymentMethodId: newPaymentMethod.id });
       }
 
+      // amount
+      if (amount !== order.totalAmount) {
+        if (amount < 100) {
+          throw new Error('Invalid amount.');
+        }
+
+        // If using a tier, amount can never be less than the minimum amount, or else $1.00
+        if (amount < tier.minimumAmount) {
```

Comment:
Have you copied something existing? What if `tier.minimumAmount` is null ?

Reply:
If we change tiers, it this the amount of the new tier or the old tier?

Reply:
It's the amount of the tier we are trying to update to.

`tier.minimumAmount` in this instance will either be the actual minimum amount for the tier, if there is one, or else it is set to `100` (or $1.00) since that's the minimum we allow anyways